mypy_self_check.ini

```diff
@@ -4,56 +4,178 @@ disallow_subclassing_any = True
 warn_no_return = True
 strict_optional = True
 no_implicit_optional = True
+disallow_any = generics, unimported, expr
```

Comment:
Wouldn't it be easier to turn on only ``generics`` and ``unimported`` now (so that there will not be many exceptions below), and then turn on ``expr`` in a separate PR?